src/encoded/types/functional_characterization_experiment.py

```diff
@@ -215,3 +216,21 @@ def crispr_screen_readout(self, request, assay_term_name, examined_loci=None, co
                 if len(set(methods)) == 1:
                     crispr_screen_readout = str(methods[0])
             return crispr_screen_readout
+
+    @calculated_property(schema={
+        "title": "Datapoint",
+        "description": "A flag to indicate whether the FC Experiment is a datapoint that should be displayed only as a part of Series object.",
+        "type": "boolean",
+        "notSubmittable": True,
+    })
+    def datapoint(self, request, related_series=None):
+        if not related_series:
+            return False
+        for series in related_series:
+            properties = {'series': series}
+            path = Path('series', include=["@type"])
+            path.expand(request, properties)
+            types = properties.get('series', {}).get('@type', [])
+            if "FunctionalCharacterizationSeries" in types:
+                return True 
+        return False
```

Comment:
Side note:  there is a EOF character on the final line, this would not pass most linters

Reply:
Question 1:  Are we going to attempt to exclude controls?  I think we shouldn't for now, but we may end up getting pushback.  

Question 2: Since we are going to likely add different types of series into this list, should we consider making it a for loop of 1 to prepare for future additions?

Reply:
A1. controls are excluded in the URL (explicitly control_type!=*) and if they on top of that belongs to series - they will be datapoints
A2. I think we will make it a loop when we have more series. I don't think we should write the code in loops just to support potential future development

Reply:
and I added an empty line 237

Reply:
as for the "datapoint vs is_datapoint" I have no strong feelings about it. I don't think we have been consistent about it: file.restricted vs biosample.is_part_of but at the same time biosample.perturbed

I think fcc.datapoint will work fine for now